### ISSUE TITLE
Drag & drop to deploy on Mac app gives "upgrade to v2" error.

### ISSUE THREAD

**Issue by georgemaine** [vercel/now-desktop#607]
When I drag & drop an item onto the mac app to deploy, it eventually gives me an error that says I should upgrade to v2.0. This gives me the idea that it failed. 

However, when I click an event in the menu bar app, it does lead me to an URL that shows the site I want to see.

I'll attach a screen recording of the steps I took. Hope that this helps!
[Archive.zip](https://github.com/zeit/now-desktop/files/2943589/Archive.zip)

**Comment by thekaeway** [comment]
I'm having the same issue!

**Comment by MAPESO** [comment]
@georgemaine Hi 👋 😄 could give me more details, it would be useful to know the version of `now-desktop` and know the configuration of your `now.json`

**Comment by thekaeway** [comment]
Do we need a now.json for single file images?

**Comment by MAPESO** [comment]
@thekaeway  It is not necessary :)

**Comment by giraffesyo** [comment]
So I'm running into this problem too. My account was created after now v1 was already unavailable. 

Basically if I drop a single file onto the tray icon, it starts to try and deploy:

<img width="382" alt="Screen Shot 2019-04-08 at 5 08 26 PM" src="https://user-images.githubusercontent.com/10459772/55760497-e726a180-5a21-11e9-85e5-7b497f6a167e.png">

Then after a while it gives this error:

<img width="488" alt="Screen Shot 2019-04-08 at 5 08 48 PM" src="https://user-images.githubusercontent.com/10459772/55760508-ef7edc80-5a21-11e9-9991-6ee1c9756189.png">

I saw that it should work because of the static builder per https://zeit.co/docs/v2/deployments/official-builders/static-now-static

So I tried it with the cli and it worked just fine.

<img width="507" alt="Screen Shot 2019-04-08 at 5 13 22 PM" src="https://user-images.githubusercontent.com/10459772/55760544-032a4300-5a22-11e9-9982-a5f180ce0e08.png">

Seems like its just a problem with the GUI not using the new builder.
